model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -46,7 +46,7 @@ data class PackageCurationData(
 
     /**
      * The concluded license as an [SpdxExpression]. It can be used to correct the license of a package in case the
-     * [declaredLicenses] found in the packages metadata or the licenses detected by a scanner do not match reality.
+     * declared licenses found in the packages metadata or the licenses detected by a scanner do not match reality.
```

Comment:
> The KDoc comment actually references the declared license of a package

I believe the comment intents to do that. Obviously it doesn't due to the brackets.

Reply:
Ah, I was reading your commit message in reverse. How about rewording as follows:

```
PackageCurationsData: Remove an unintended reference to declaredLicenses in docs

The KDoc comment is supposed to refer to the declared licenses of a
package, not the `declaredLicenses` field of `PackageCurationsData` itself.
So reword the sentence to not use a reference link at all.```

Reply:
Or better yet, just fix the link (and text):

```
    /**
     * The concluded license as an [SpdxExpression]. It can be used to correct the [declared licenses of a package]
     * [Package.declaredLicenses] in case the found in the packages metadata or the licenses detected by a scanner
     * do not match reality.
     */
```